tests/test_restamp_testflight_archive.py

```
@@ -0,0 +1,178 @@
+        archive, app = make_archive(self.tmp_path)
+
+        MODULE.validate(archive, "dev.cmux.app.dev", "cmux DEV", "1.0.4")
+
+        with (app / "Info.plist").open("rb") as handle:
+            app_info = plistlib.load(handle)
+        self.assertEqual(app_info["CFBundleVersion"], "10")
+
+    def test_extracts_tar_safely_and_preserves_executable_mode(self) -> None:
+        source_archive, _ = make_archive(self.tmp_path / "source", "cmux-pr-demo.xcarchive")
+        tar_path = self.tmp_path / "archive.tar"
+        subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)